### PR TITLE
Remove first prefix from submenus in library reference

### PR DESCRIPTION
**Pull request by JinShil**
no_std

### CODE REVIEW

**Comment by andralex** [comment]
I think we should move forward with this, but just to make sure I've asked around. http://forum.dlang.org/thread/ma0kuu$1f08$1@digitalmars.com#post-ma0kuu:241f08:241:40digitalmars.com

**Comment by CyberShadow** [comment]
@andralex If it's not too hard, please use the new "Permalink" links (by right-clicking them) to share forum URLs. http://forum.dlang.org/help#canonical

**Comment by quickfur** [comment]
Yes, please!

What about std.range.* and std.container.*? IMO those should also be collapsed, i.e., std.container.* should be collapsed to a single item, expandable to array, list, rbtree, etc..

**Comment by andralex** [comment]
looks like this is good to go. please rebase

**Comment by JinShil** [comment]
I rebased, but I have no idea if I did it right.  It looks like everything's fine to my untrained eye, but please apply scrutiny.

File: std_navbar-2.066.1.ddoc [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
+LIBITEM=$(A $1$(LIBITEM_RECURSE1 $+).html, $(D $2))
+LIBITEM2=$(A $1$(LIBITEM_RECURSE1 $+).html, $(D $(LIBITEM_RECURSE2 $+)))
 LIBITEM_RECURSE1=_$1$(LIBITEM_RECURSE1 $+)
+<<<<<<< HEAD
+LIBITEM_RECURSE2=$1$(LIBITEM_RECURSE3 $+)
+LIBITEM_RECURSE3=.&#x200B;$1$(LIBITEM_RECURSE3 $+)
+=======
 LIBITEM_RECURSE2=.&#x200B;$1$(LIBITEM_RECURSE2 $+)
+>>>>>>> 870183cbd442b6af8e57ab2cdbab1ce0659ee586
 _=
```
Comment (CyberShadow) [inline std_navbar-2.066.1.ddoc]: Nope, you need to resove this merge conflict.

**Comment by JinShil** [comment]
> What about std.range.* and std.container.*? IMO those should also be collapsed, i.e., std.container.* should be collapsed to a single item, expandable to array, list, rbtree, etc..

I agree, but that looks like a non-trivial change.  If I can get to it, I'll submit a separate pull request.

**Comment by CyberShadow** [comment]
> I agree, but that looks like a non-trivial change.

I have an idea on how to handle it.

**Comment by JinShil** [comment]
Ok, I hope we're good to go now.

**Comment by CyberShadow** [comment]
Please rebase, there is a merge commit in your commit list :)

`git rebase -i origin/master`, then change `pick` to `fixup` for every line except the first.

**Comment by CyberShadow** [comment]
> `git rebase -i origin/master`, then change `pick` to `fixup` for every line except the first.

Actually, not every commit. Just leave the commits containing *your* changes.

**Comment by JinShil** [comment]
screw it! I'm making a new pull request.

**Comment by CyberShadow** [comment]
You could have just recreated the branch...

**Comment by CyberShadow** [comment]
> I have an idea on how to handle it.

https://github.com/D-Programming-Language/dlang.org/pull/841

**Pull request closed without merging**